packages/freshmint/commands/burn.ts

```diff
@@ -0,0 +1,35 @@
+import { Command } from 'commander';
+import ora from 'ora';
+import chalk from 'chalk';
+
+import { FlowGateway, FlowNetwork } from '../flow';
+import { FreshmintError } from '../errors';
+import { loadConfig } from '../config';
+import { parseUFix64Argument } from '../arguments';
+
+export default new Command('burn')
+  .argument('<ids>', 'The IDs of NFTs to destroy (e.g. 3425 1235 4524 216661).', parseUFix64Argument)
```

Comment:
```suggestion
  .argument('<ids...>', 'The IDs of NFTs to destroy (e.g. 3425 1235 4524 216661).', parseUFix64Argument)
```

I think `ids` should be a variadic argument as per the [commander.js documentation](https://github.com/tj/commander.js/#command-arguments)